### PR TITLE
Cleanup of out of date firmware references

### PR DESCRIPTION
**Pull request by Hwurzburg**
Start of wiki cleanup of outdated firmware reference. Per previous agreements only references to last two releases in a vehicle will be retained. Archived topics are exempt. And where large pieces of information would be removed, they will be archived in some manner instead of deleted.

### CODE REVIEW

File: copter/source/docs/gps-failsafe-glitch-protection.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ A *GPS failsafe* event will occur if GPS 3D
 lock or the position "Glitches" for at least 5 seconds while Copter is
 in a mode that requires the GPS (RTL, Auto, Loiter, Circle, Position,
 Guided or Drift).  The GPS failsafe response can be set to Land or
-switch to AltHold mode (in AC3.1 and higher) so that you can retake
+switch to AltHold mode so that you can retake
```
Comment (rmackay9) [inline copter/source/docs/gps-failsafe-glitch-protection.rst:23]: I think this page is perhaps not linked from anywhere anymore.  Not a problem to update it but I don't see how users can get to it.  It should probably be archived if it's not already.

**Comment by rmackay9** [comment]
Merged, thanks!

File: copter/source/docs/gps-failsafe-glitch-protection.rst (hunk 1 above)
Comment (Hwurzburg) [reply copter/source/docs/gps-failsafe-glitch-protection.rst:23]: it is....from EKF failsafe....and an unlinked page generates a build error...
are you sure that the DCM and EKF GPS glitch protection talked about on this page is no longer used?

Comment (rmackay9) [reply copter/source/docs/gps-failsafe-glitch-protection.rst:23]: This page (https://ardupilot.org/copter/docs/gps-failsafe-glitch-protection.html) is linked from the bottom of the EKF failsafe page but it was for the gps glitch protection that built into copter and the EKF1 (which we don't use anymore). It's possible the EKF1 section could be moved to the EKF failsafe page and updated to apply to the EKF2/EKF3.  It seems that:

- EKF_GLITCH_RAD has become EK2_GLITCH_RAD and EK3_GLITCH_RAD
- EKF_POS_GATE has become EK2_POS_I_GATE, EK3_POS_I_GATE
- GPSGLITCH_ACCEL doesn't exist

**Pull request merged**